docs/tutorial.md

```diff
@@ -341,7 +546,13 @@ Currently, your shopping list application has a write side that allows you to cr
 
 ### Add a Read Model
 
-Add a **ShoppingLists** **[Read Model](read-side.md#read-models)** to your application. A Read Model receives events and populates a persistent store based on event data. It uses the collected data to answer data queries.
+Add a **ShoppingLists** Read Model to your application.
+
+:::tip Read Model
+
+A Read Model is entity at the application's read side that answers data requests. A Read Model receives events and populates a persistent store based on event data. It then uses the collected data to build the requested data samples.
```

Comment:
`A Read Model is entity at the application's read side that answers data requests.`

What do you mean by "entity"?

Reply:
I mean an abstract "program entity". I rewrote the sentence to be more specific:
"A Read Model is a data model at the application's read side that answers data requests."

Reply:
Sounds good. Thank you.